README.md

```diff
@@ -6,25 +6,77 @@
 
 1. Add the following to your **root** `build.gradle` file:
 
-```
-allprojects {
-    repositories {
-        ...
+    ```
+    allprojects {
+        repositories {
+            ...
 
-        maven { url "https://raw.githubusercontent.com/placed/android-placed-sdk/master/repository" }
+            maven { url "https://raw.githubusercontent.com/placed/android-placed-sdk/master/repository" }
+        }
     }
-}
-```
+    ```
 
 2. Add the following to your **app** `build.gradle` file:
 
-```
-dependencies {
-    ...
+    ```
+    dependencies {
+        ...
+
+        compile 'com.placed.client:android-persistent-sdk:1.30'
+    }
+    ```
+
+3. If you use ProGuard, add the following to your `proguard-rules.pro` file:
```

Comment:
Is there any way to "inherit" proguard rules from an SDK like you can with AndroidManifest?